sklearn/metrics/tests/test_classification.py

```diff
@@ -3094,3 +3098,129 @@ def test_d2_log_loss_score_raises():
     err = "The labels array needs to contain at least two"
     with pytest.raises(ValueError, match=err):
         d2_log_loss_score(y_true, y_pred, labels=labels)
+
+
+def test_d2_brier_score():
```

Comment:
Could you parametrize this test with `y_pred`, `expected_score`? `y_true` seems to stay fixed.

Reply:
Yes agreed that y_true is fixed. But the first instance where we compare the brier score of the model with the ref brier score is a bit different than the other cases. So either we should move that into another test and then parametrize the cases where we are checking for expected score == 0. Or we can keep it the way it is currently.